project/api/models.py

```diff
@@ -257,6 +309,9 @@ class Narrative(models.Model):
     url = models.TextField(unique=True)
     description = models.TextField()
     tags = ArrayField(models.TextField(max_length=100))
+    votes = models.ManyToManyField(
```

Comment:
Are you sure that this field should have a Many-to-Many relationship?

Reply:
It's not exactly m2m. Multiple users can vote on multiple narratives, but the same user cannot vote on the same narrative multiple times. A m2m field is the only relation type that allows the through (intermediary) field.

Reply:
Ah, ok, so the user is separated, sounds like django magic, again. Ok then.